internal/drivers/openstack/keystone.go

```diff
@@ -149,6 +149,7 @@ func (d *keystoneDriver) AuthenticateUser(ctx context.Context, userName, passwor
 	)
 
 	if t.Err != nil {
+		//nolint:errorlint // the casted variable is used
```

Comment:
Another case where the nolint description can be clarified: `errors.As()` does support retrieving the casted error value, we just don't consider it worth the effort because it's more tedious than a straight-up type cast. Same for the other identical nolint descriptions below.